src/components/Display.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] I guess we can remove this rule here since this is already a stateless component.